data/cvr/samples/jetsons/jetsons_bedrock-precinct_cvr.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CastVoteRecordReport xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://itl.nist.gov/ns/voting/1500-103/v1 ../../../../../CastVoteRecords/NIST_V0_cast_vote_records.xsd"
+ xmlns="http://itl.nist.gov/ns/voting/1500-103/v1">
+    <CVR>
+        <BallotImage>
+            <Image FileName="bedrock_ballot_1" MimeType="application/pdf" />
```

Comment:
Use: `ballot-image-bedrock-precinct-01.pdf`?
This a filename and not an ID, so arguably the convention can be different. But I'd like to make them visually distinct so if not that then `Bedrock_Ballot_01.pdf` would be better.